libethereum/Account.cpp

```diff
@@ -119,12 +100,12 @@ AccountMap dev::eth::jsonToAccountMap(std::string const& _json, u256 const& _def
 	js::mValue val;
 	json_spirit::read_string_or_throw(_json, val);
 	js::mObject o = val.get_obj();
-	validateAccountMapObj(o);
-	for (auto const& account: o)
+    for (auto const& account: o)
 	{
-		Address a(fromHex(account.first));
+        Address a(fromHex(account.first));
 		// FIXME: Do not copy every account object.
 		auto o = account.second.get_obj();
+        validateAccountMapObj(o);
```

Comment:
Why do you do this now inside the loop? Validating the whole map object in each iteration?

Reply:
checking one account instead of a list of accounts

Reply:
Ah wow, don't use `o` for both the whole map and one account object.
Use more meaningful names than `o` anyway